packaging/linux/openra.metainfo.xml.in

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
+<component type="desktop-application">
   <id>openra-{MODID}.desktop</id>
```

Comment:
> The ID must follow a reverse-DNS scheme, consisting of {tld}.{vendor}.{product}, for example org.kde.gwenview or com.hugski.colorhug2. Ownership of {vendor}.{tld} in the domain name system guarantees uniqueness of IDs.

and 

>  Note that even though the component-ID can now be any rDNS name, when updating existing applications, do not change their <id/> to drop the .desktop suffix.

These conflicting requirements are a bit awkward, since this metainfo template is used for both existing projects and new projects based on the Mod SDK. Oh well...

Reply:
I think the second note is to ensure backwards compatibility. Dropping the .desktop suffix makes it effectively a different ID (which should be avoided without good reasons).

Reply:
Changing the id is not in the scope of this PR. First you must agree on a new id and probably coordinate the change with packagers.

Reply:
Ah, I wasn't asking to change this. Just commenting on the conflicting requirements that the spec change has created.

Reply:
On @FlatHub they changed it https://github.com/flathub/net.openra.OpenRA/blob/master/openra.metainfo.xml.in#L3